test/lib/instruction-parser.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { getPackagesFromRunInstructions } from '../../lib/instruction-parser';
+
+describe('instruction parsers', () => {
```

Comment:
Can we add another test for an install command spitted by multiple line?
`RUN apt-get update && apt-get install -y \
    python-qt4 \
    python3-pyqt5`

Reply:
Ah good suggestion! I don't think my code catches this.